lib/runtime-tools/heaptrack-wrapper.ts

```diff
@@ -169,7 +171,7 @@ export class HeaptrackWrapper extends BaseRuntimeTool {
 
         await this.makePipe();
 
-        const fd = fs.openSync(this.pipe, O_NONBLOCK | O_RDWR);
+        const fd = fs.openSync(this.pipe, O_NONBLOCK | fsConstants.constants.O_RDWR);
```

Comment:
That's weird. I didn't get this locally, and somehow this doesn't apply to `O_NONBLOCK`?  Could this be some glitch?

Reply:
O_NONBLOCK is a local constant in this file because it doesn't exist on some platforms/nodeversions

Reply:
Maybe you intended to replace `import {O_RDWR} from 'constants';` into something else?

Reply:
yes, the linter considers constants to be deprecated and you should use the module specific constants

Reply:
So you need something like 
```ts
import {constants as fsConstants} from 'node:fs';
.
.
.
const fd = fs.openSync(this.pipe, O_NONBLOCK | fsConstants.O_RDWR);
```

No?

Reply:
yes, thank you